src/Zinc-HTTP/ZnClient.class.st

```diff
@@ -915,9 +915,9 @@ ZnClient >> maxNumberOfRedirects [
 	"Return the maximum number of HTTP redirect that will be followed.
 	Note that when the #followsRedirect option is false, 
 	no redirects will ever be followed regardless of the value of this option.
-	Defaults to 3. A ZnTooManyRedirects will be signalled when the count reaches zero."
+	Defaults to 5. A ZnTooManyRedirects will be signalled when the count reaches zero."
 
-	^ self optionAt: #maxNumberOfRedirects ifAbsent: [ 3 ]
+	^ self optionAt: #maxNumberOfRedirects ifAbsent: [ 5 ]
```

Comment:
We should not change the global redirect to fix a test. The test should be modified, this has a really bad impact.
Has this change been impacted in Pharo10?

Reply:
This is the fix that was done in the Zinc upstream. It is already in Pharo10

Reply:
see the discussion in https://github.com/pharo-project/pharo/issues/10290

Reply:
> We should not change the global redirect to fix a test. The test should be modified, this has a really bad impact.
> Has this change been impacted in Pharo10?

3 was a bit too low, in certain valid cases, like in the test, more than 3 redirects do happen.
The main objective is to prevent infinite loops, as these do happen occasionally.

Reply:
Ok, perfect